sdk/identity/identity/rollup.base.config.js

```diff
@@ -87,7 +87,7 @@ export function browserConfig(test = false) {
   };
 
   if (test) {
-    baseConfig.input = "dist-esm/test/internal/*.spec.js";
+    baseConfig.input = ["dist-esm/test/**/*.spec.js"];
```

Comment:
Why this was only internal before is because many of the _public_ tests are only supposed to work in Node. I will figure a way around this as part of this PR.